contents/stats_vis.md

```diff
@@ -1,7 +1,224 @@
 ## Statistical Visualizations {#sec:stats_vis}
 
+There are several statistical visualization techniques.
+We will focus on mainly two: **box plots** and **density plots**, since they are the most used and our preferred way to analyze univariate data.
+
+We will also use the `more_grades` dataset from @sec:stats_central.
+
 ### Box Plots {#sec:stats_vis_boxplots}
 
+Box plots are a method for graphically depicting numerical data through their quartiles (see @fig:boxplot).
+The "box" is typically represented by the quartiles 1 to 3.
```

Comment:
Probably, saying what a quartile is would not hurt here.

Reply:
Actually they are already in the early part of the chapter. I will put something like:
> (see section XX)